plugins/providers/gcloudiam/provider.go

```diff
@@ -34,6 +36,16 @@ func (p *Provider) CreateConfig(pc *domain.ProviderConfig) error {
 		return err
 	}
 
+	client, err := p.getIamClient(pc)
+	if err != nil {
+		return err
+	}
+
+	r := c.ProviderConfig.Resources[0]
```

Comment:
even though we will always have one resource and it's also already validated before it goes here, let's still use `for _, r := range c.ProviderConfig.Resources` to avoid confusion and `index out of range error` if there's a change in the future

Reply:
Done